sql/testdata/explain_debug

```diff
@@ -3,9 +3,10 @@ CREATE TABLE abc (
   a INT,
   b TEXT,
   c FLOAT,
+  d CHAR,
   PRIMARY KEY (a, b),
-  CONSTRAINT foo UNIQUE (b),
-  CONSTRAINT bar INDEX (a)
+  CONSTRAINT foo UNIQUE (c),
```

Comment:
indexing of floats was never the problem - writing NULL to any indexed column is the problem case, which i decided to side-step in this PR